monitoring/managing-metrics-targets.adoc

```diff
@@ -0,0 +1,30 @@
+[id="managing-metrics-targets"]
+= Managing metrics targets
+include::modules/common-attributes.adoc[]
+:context: managing-metrics-targets
+
+toc::[]
+
+{product-title} Monitoring collects metrics from targeted cluster components by scraping data from exposed service endpoints. 
+
+In the Administrator perspective in the {product-title} web console, you can use the *Metrics Targets* page to view, search, and filter the endpoints that are currently targeted for scraping. 
```

Comment:
What's the benefit of scraping? Does scraping omit what is unnecessary, therefore, quickening the development process? If so maybe, consider a brief explanation of the benefit of scraping. Something like, 

... you can use the **Metrics Targets** page to view, search, and filter the endpoints that are currently targeted for scraping. When you know what endpoints are targeted for scraping, you then can select only the endpoints you need for your development purposes.

Reply:
Mainly it's helping you to identify and troubleshoot problematic endpoints.

Reply:
Alright, cool, again, it's definitely good to leave as is. If you'd like, you could say what you just told me:

..you can use the **Metrics Targets** page to view, search, and filter the endpoints that are currently targeted for scraping which helps you to identify and troubleshoot problematic endpoints.

Reply:
I like that and will add it. Thanks @jc-berger!